lib/registries/addon/drafts/draft/draft-registration-manager.ts

```diff
@@ -21,9 +22,15 @@ import {
 } from 'ember-osf-web/packages/registration-schema';
 import buildChangeset from 'ember-osf-web/utils/build-changeset';
 
+type DraftsDraftModelTaskInstance = TaskInstance<{
```

Comment:
Can we change this to something a bit more readable?

Reply:
Done!